.github/workflows/r-test.yml

```diff
@@ -0,0 +1,74 @@
+name: testthat
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        
+      - name: Install Ubuntu dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libcurl4-openssl-dev libharfbuzz-dev libfribidi-dev
+
+      - name: Install R dependencies
+        run: |
+          install.packages('remotes')
+          install.packages('devtools')
+          remotes::install_deps(dependencies = TRUE)
+          remotes::install_cran("rcmdcheck")
+          remotes::install_cran("httr")
+          remotes::install_github("Sage-Bionetworks/dccvalidator")
+          devtools::install()
```

Comment:
This is fine, but I wonder if we could use `renv` so there's a maintenance on specific versions that are installed as well.

Reply:
Good point! i'll make that update